src/components/todo.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import TodoList from "./todolist";
+// your components will all go in this `component` directory.
+// feel free to change this component.js into TodoList.js
+
+class Todo extends Component {
+    constructor() {
+        super();
+        this.state = {
+        todos: "",
+        todo: "",
+        
+        }
+    };
+
+
+    nameReciever = (event) => {
+        this.setState({ [event.target.description]: event.target.value });
+    }; 
+    
+    handleSunmitTodo = () => {
+        const todos = this.state.todos;
+        const todo = { name: this.state.todo, id:this.state.todo };
+        todos.push(todo);
+        this.setState({ todos: todos, todo: "" });
+    };
+
+
+
+    render() {
+        return (
+        <div>
+         Todo 
+          <todos todo={this.state.todo} />   
```

Comment:
In this line the class todo does not exist. Not sure what you intended to implement here, but my assumption is that you intended to display the this.state.todo as text inside a div. To accomplish this try this lines:

```<div>{this.state.todo}</div>```